### ISSUE TITLE
Omitting visualLevel parameter causes JSONException on Android

### ISSUE THREAD

**Issue by TheStalwart** [OneSignal/OneSignal-Cordova-SDK#133]
Calling setLogLevel with only one parameter works on iOS, but crashes on Android:
`window.plugins.OneSignal.setLogLevel({logLevel: 4});`

```
             System.err  W  org.json.JSONException: No value for visualLevel
                         W      at org.json.JSONObject.get(JSONObject.java:389)
                         W      at org.json.JSONObject.getInt(JSONObject.java:478)
                         W      at com.plugin.gcm.OneSignalPush.execute(OneSignalPush.java:266)
                         W      at org.apache.cordova.CordovaPlugin.execute(CordovaPlugin.java:98)
                         W      at org.apache.cordova.PluginManager.exec(PluginManager.java:132)
                         W      at org.apache.cordova.CordovaBridge.jsExec(CordovaBridge.java:57)
                         W      at org.apache.cordova.engine.SystemExposedJsApi.exec(SystemExposedJsApi.java:41)
                         W      at org.chromium.base.SystemMessageHandler.nativeDoRunLoopOnce(Native Method)
                         W      at org.chromium.base.SystemMessageHandler.handleMessage(SystemMessageHandler.java:41)
                         W      at android.os.Handler.dispatchMessage(Handler.java:102)
                         W      at android.os.Looper.loop(Looper.java:211)
                         W      at android.os.HandlerThread.run(HandlerThread.java:61)
  SystemWebChromeClient  D  file:///android_asset/www/js/index.js: Line 518 : Uncaught TypeError: window.plugins.OneSignal.startInit is not a fu
                            nction
```

**Comment by jkasten2** [comment]
@TheStalwart Thanks for the report, we will look into making this compatible with Android as well in a future update.

**Comment by jkasten2** [comment]
@TheStalwart This is now fixed in the 2.1.0 version of this SDK.
